genesis/model/resolver/doc.go

```diff
@@ -43,5 +43,16 @@ Usage:
 	proxy.GetParent() always returns nil.
 	proxy.GetClassID() is a proxy call for instance method.
 
+
+ReferenceContainer provides methods for store Reference as BaseProxy
```

Comment:
store Reference as **_Proxy_**
(because Proxy is a common interface, and BaseProxy just an implementation)

Reply:
OK